capstone/client/src/components/StoresPage.js

```diff
@@ -73,7 +74,8 @@ class StorePage extends Component {
           console.log(res.data);
           this.setState({
             stores: res.data,
-            originalStores : res.data
+            originalStores : res.data,
+            storesFound : true,
           });
         });
```

Comment:
Probably should also catch the API error and show the no stores found page as well